imagebuildah/stage_executor.go

```diff
@@ -2581,3 +2600,32 @@ func (s *StageExecutor) EnsureContainerPathAs(path, user string, mode *os.FileMo
 	logrus.Debugf("EnsureContainerPath %q (owner %q, mode %o) in %q", path, user, mode, s.builder.ContainerID)
 	return s.builder.EnsureContainerPathAs(path, user, mode)
 }
+
+func (s *StageExecutor) Annotations(isLastStep bool) string {
```

Comment:
This method should not be exported.  Naming it `Annotations`, when it isn't solely about them, is potentially confusing to other maintainers.